src/scss/main.scss

```diff
@@ -977,3 +977,13 @@ input[type='color'] {
 .input-border {
     border: 1px solid #ced4da;
 }
+
+.template-spacing {
```

Comment:
don't add a class like that, use "mt-2" class. See: https://getbootstrap.com/docs/4.3/utilities/spacing/

Reply:
Removed this class in the recent commit.